controllers/channel.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const mysql   = require('anytv-node-mysql');
+const winston = require('winston');
+const util = require(__dirname + '/../helpers/util');
+
+
+exports.get_channels = (req, res, next) => {
+    let connection = mysql.use('dashboard_db');
```

Comment:
and you dont really need to do this. you can just not end a mysql.query() and do it again :)